### DISCUSSION TITLE
Unable to find Docker Image with Bugfix

### DISCUSSION THREAD

**Discussion by lindsayrblank** [dtcenter/METplus#1899] [Existing Builds] [answered]
Hello @JohnHalleyGotway,

In our previous discussion ( #1826 ), we had talked about a MET 10.0 docker image that included the bugfix. I can't seem to find a MET 10.0 docker image that is more recent than December 2021 on the DTC Docker page. Thanks for your help!

**Comment by JohnHalleyGotway** [comment, marked as answer]
Hi @lindsayrblank,

I see you'd like to get access to a version of MET that contains the bugfix for the [dtcenter/MET#2271](https://github.com/dtcenter/MET/issues/2271) issue.

Under the "Development" section on the right side of that issue, I see that it's linked to 3 different pull requests. That means we fixed that bug in 3 different spots:
1. Pull request [dtcenter/MET#2272](https://github.com/dtcenter/MET/pull/2272) fixed it in the develop branch so that it'll be included in future releases.
2. Pull request [dtcenter/MET#2274](https://github.com/dtcenter/MET/pull/2274) fixed it in the main_v10.1 branch so that it'll be included in v10.1 bugfix releases.
3. Pull request [dtcenter/MET#2273](https://github.com/dtcenter/MET/pull/2273) fixed it in the main_v10.0 branch so that it'll be included in v10.0 bugfix releases.

And you're looking for a v10.0 bugfix. Clicking the [dtcenter/MET#2273](https://github.com/dtcenter/MET/pull/2273) PR, I see that it's assigned to the "10.0.2 (bugfix)" milestone. However, checking the [MET Releases](https://github.com/dtcenter/MET/releases), we haven't actually created that bugfix release!

So it looks to me like we need to cut that v10.0.2 bugfix release to make this available to you.

I'll plan to do that tomorrow, and that should automatically be built on DockerHub and become available via:
```
docker pull dtcenter/met:10.0.2
```

Does that sound good to you?

**Reply by lindsayrblank** [reply]
That sounds great, thank you!

**Reply by JohnHalleyGotway** [reply]
OK Lindsay, the MET 10.0.2 is created, and [DockerHub](https://hub.docker.com/repository/registry-1.docker.io/dtcenter/met/builds/1b6297da-1908-43a7-9fb5-35c15ad6158d) built it successfully for us.

You can see the release on [GitHub](https://github.com/dtcenter/MET/releases/tag/v10.0.2) and also announced on the [DTC website](https://dtcenter.org/community-code/metplus/met-version-10-0-2).

Please try running:
```
docker pull dtcenter/met:10.0.2
```
Note that the following also works:
```
docker pull dtcenter/met:10.0-latest
```
That pulls the most recent bugfix release for that X.Y version. Might not matter here, but could be useful for future releases.
